README.md

```diff
@@ -350,3 +377,4 @@ See [CONTRIBUTING.md](CONTRIBUTING.md) for details.
 ## License
 
 See [LICENSE](LICENSE) for details.
+````
```

Comment:
This looks extraneous

Reply:
that may have been prettier. I will check.

Reply:
removed.